mail_auto_resend/models/mail_mail.py

```diff
@@ -13,7 +13,6 @@ def _cron_resend_failed_mails(self):
         failed_mails = self.search([("state", "=", "exception")])
         try:
             failed_mails.write({"state": "outgoing"})
-            self.env.cr.commit()
             _logger.debug("Batch of previously failed emails are now outgoing")
         except Exception:
             self.env.cr.rollback()
```

Comment:
not sure that the rollback is necessary

Reply:
It's not: it's only necessary when you're using a cursor you created yourself.